documentation/modules/proc-creating-configmap.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// assembly-external-logging.adoc
+
+[id='creating-configmap_{context}']
+= Creating a ConfigMap for logging
+
+To use a ConfigMap to define logging properties, you create the ConfigMap and then reference it as part of the logging definition in the `spec` of a resource.
+
+The ConfigMap must contain the appropriate logging configuration.
+
+* `log4j.properties` for Kafka components
+* `log4j2.properties` for the Topic Operator and User Operator
+
+Here we demonstrate how a ConfigMap defines a root logger for a Kafka resource.
+
+.Procedure
+
+. Create the ConfigMap.
++
+You can create the ConfigMap as a YAML file or from a properties file at the `kubectl` command line.
++
+ConfgMap example with a root logger definition for Kafka:
++
+[source,yaml,subs="+attributes"]
+----
+kind: ConfigMap
+apiVersion: {KafkaApiVersion}
+metadata:
+  name: logging-configmap
+data:
+  log4j.properties:
+  # logger settings for resource
+    kafka.root.logger: "INFO"
```

Comment:
the syntax in a properties file is with "=" not ":" so it should be 
kafka.root.logger=INFO